lib/rubocop/cop/layout/indent_heredoc.rb

```diff
@@ -37,7 +37,7 @@ module Layout
       #   # good
       #   # When EnforcedStyle is powerpack, bad code is auto-corrected to
       #   # the following code.
-      #   <<~RUBY
+      #   <<~RUBY.strip_indent
```

Comment:
The heredoc should be normal here, not a squigly one.

Reply:
Oh, right. Fixed.

Reply:
This squiggly heredoc is a mistake that was included below. Thanks for the fixing.
https://github.com/rubocop-hq/rubocop/commit/5ffddbb92926122a4f3fc1bf8ae29d5be31d481f